service/controller/v17patch2/version_bundle.go

```diff
@@ -8,8 +8,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "containerlinux",
-				Description: "Update to 1967.5.0 (CVE-2019-5736).",
+				Component:   "kubernetes",
+				Description: "Update to 1.12.6 (CVE-2019-1002100).",
 				Kind:        versionbundle.KindChanged,
```

Comment:
you want to add new changelog entry instead if editing already existing

Reply:
This comes from the full copy of process of v17patch1.
So that entry belongs to v17patch1 and I guess it is not required in v17patch2. Isn´t it?

Reply:
right,  i thought that you are rewriting the new coreos